CMakeLists.txt

```
@@ -44,7 +44,22 @@ if(BUILD_MCNP5 OR BUILD_MCNP6)
   enable_language(Fortran)
 endif()
 
-find_package(MOAB REQUIRED)
+# Dealing with MOAB
+if (PULL_INSTALL_MOAB)
+  set(MOAB_VERSION ${PULL_INSTALL_MOAB})  
+
+  # Ensure MOAB_VERSION
+  if(NOT ${MOAB_VERSION} MATCHES "^[0-9]+\\.[0-9]+\\.[0-9]")
```

This should address @shimwell's latest question
```suggestion
  if(NOT ${MOAB_VERSION} MATCHES "^[0-9]+\\.[0-9]+\\.[0-9]+")
```

---

It is now broken I don't understand why...
I'll add this suggestion to the branch as soon I understand why it is not working anymore